proctoru/proctoru.py

```diff
@@ -373,7 +373,8 @@ def student_view(self, context=None):
 
                     allowed_keep_old_exam = False
 
-                    if exam_start_date_time <= old_exam_time <= exam_end_date_time:
+                    if old_exam_time.tzinfo is not None  \
+                            and (exam_start_date_time <= old_exam_time <= exam_end_date_time):
```

Comment:
Python will stop evaluating when first condition is False when using `and` operator